app/src/main/java/edu/uc/clemenks/keepingupwithschool/ui/main/MainFragment.kt

```diff
@@ -54,6 +54,7 @@ class MainFragment : Fragment() {
 
     }
 
+    //Checks permission request for access to camera
```

Comment:
I added this comment for this method to keep method commenting continuity intact.